tests/test_open_graph.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import networkx as nx
+
+from graphix.open_graph import Measurement, OpenGraph
+from graphix.pauli import Plane
+
+
+# Tests whether an open graph can be converted to and from a pattern and be
+# successfully reconstructed.
+def test_open_graph_to_pattern() -> None:
```

Comment:
We should enrich `random_objects.py` with random open graph generation (possible naive to begin with), and test against it. We could also add a test with random patterns (for instance, transpiled from random circuits).

Reply:
Good idea.
This is a non-trivial task and so I think this would go better in another PR perhaps. I've made an issue to track it if thats fine with you? #196